fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/FedoraResourceImpl.java

```diff
@@ -192,6 +192,11 @@ protected FedoraResource doBackward(final FedoraResource fedoraResource) {
             }
             return fedoraResource;
         }
+
+        @Override
```

Comment:
Not supporting `hashCode()` seems like an a sign of a more fundamental issue.
But any ways, the related Sonar issue seems to be gone: http://sonar.fcrepo.org/drilldown/issues/1?&rule=findbugs%3AEQ_COMPARETO_USE_OBJECT_EQUALS&rule_sev=MAJOR&severity=MAJOR